libs/frontend/util-router/src/lib/scroll-behavior.ts

```diff
@@ -20,20 +20,6 @@ export interface ScrollPositionElement extends ScrollToOptions {
 
 export type ScrollPosition = ScrollPositionCoordinates | ScrollPositionElement
 
-function getElementPosition(
-  el: Element,
-  offset: ScrollPositionCoordinates,
-): _ScrollPositionNormalized {
-  const docRect = document.documentElement.getBoundingClientRect()
-  const elRect = el.getBoundingClientRect()
-
-  return {
-    behavior: offset.behavior,
```

Comment:
Won't removing this function break the situation where `behavior` has been set? Maybe I'm missing something.

Reply:
Oh, I removed it because I only found one occurrence of `getElementPosition` in the workspace, and that usage is being replaced by `element.scrollIntoView()` by me in this PR.
Is it actually a function that gets called somewhere else?

Reply:
No, I was just curious if the new way of using scrollIntoView covers all the same usecases of getElementPosition. It should be fine!